internal/transform/trade.go

```diff
@@ -215,3 +233,32 @@ func extractClaimedOffers(operationResults []xdr.OperationResult, operationIndex
 
 	return
 }
+
+func getClaimedOrderBook(offers []xdr.ClaimAtom) (orderBookOffers []xdr.ClaimOfferAtom, err error) {
+	var singleClaimOfferAtom xdr.ClaimOfferAtom
+	for _, singleOffer := range offers {
+		switch singleOffer.Type {
+		case xdr.ClaimAtomTypeClaimAtomTypeV0:
+			// Protocols 17 and 18 changes the Orderbook structure
+			singleOfferOrders := singleOffer.MustV0()
```

Comment:
I actually don't know if we're ok with the panic. It'll be safer to use `GetV0` instead so that we don't cause interruptions in the ETL if there is an invalid value.